src/plugins/controlbox/styles/controlbox-head.scss

```diff
@@ -5,7 +5,7 @@
             flex-direction: row-reverse;
             flex-wrap: nowrap;
             justify-content: space-between;
-            min-height: 0;
+            min-height: 30px;
```

Comment:
Why was this necessary?

Reply:
sorry i haven't check well i forgot to check properly now it's not needed to set min-height to 30px because the chat box is resizable box so it won't affect